modules/ztp-validating-the-generation-of-installation-crs.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-validating-the-generation-of-installation-crs_{context}"]
+= Validating the generation of installation CRs
+
+SiteConfig applies Installation custom resources (CRs) to the hub cluster in a namespace with the name matching the site name. To check the status, use the following command:
+
+[source,terminal]
+----
+$ oc get AgentClusterInstall -n <clusterName>
+----
+
+If no object is returned, use the following procedure to troubleshoot the ArgoCD pipeline flow from SiteConfig to the installation CRs.
+
+.Procedure
+
+. Check the synchronization of the SiteConfig to the hub cluster using either of the following commands:
++
+[source,terminal]
+----
+$ oc get siteconfig -A
+----
++
+or
++
+[source,terminal]
+----
+$ oc get siteconfig -n clusters-sub
+----
++
+If the SiteConfig is missing, one of the following situations has occurred:
+
+* The *clusters* application failed to synchronize the CR from the Git repository to the hub. Use the following command to verify this:
++
+[source,terminal]
+----
+$ oc describe -n openshift-gitops application clusters
+----
++
+Check for `Status: Synced` and that the `Revision:` is the SHA of the commit you pushed to the subscribed repository.
++
+* The pre-sync hook failed, possibly due to a failure to pull the container image.
```

Comment:
Is there a way to avoid "possibly" here?

Reply:
Not really.